lib/parser.js

```diff
@@ -22,10 +17,10 @@ const constants = {
 
 const readParcel = (
   // Read parcel from buffer
-  buffer // [buffer type]
-  // Return parcel object
+  buffer // buffer
+  // Return: object
 ) => {
-  const rest = Buffer.from(buffer, 0, PARCEL_LENGTH);
+  const rest = Buffer.from(buffer, 0, constants.PARCEL_LENGTH);
```

Comment:
If `buffer` is `Buffer` type, than why do you need last 2 arguments? Moreover, since you only read from it, there is no sense in additional memory copying. Looking at arguments, I can assume that this is `BufferArray`, but in this case `readParcel`s' interface comments are not correct. Please, read [Buffer documentation](https://nodejs.org/api/buffer.html) carefully and make corrections according to this.